### DISCUSSION TITLE
AMD Support

### DISCUSSION THREAD

**Discussion by ai-d-g** [deforum-art/deforum-stable-diffusion#336] [Ideas]
I was curious if the 3D aspect of Deforum to graphics drivers / hardware resides in "py3d_tools.py" only  ? Curious if changing that to work with AMD would be possible or is there is a limitation on Torch with AMD hardware ?

**Comment by ai-d-g** [comment]
I reinstalled the latest Automatic1111 build and the 3D now works on AMD. 
ARGS = --skip-torch-cuda-test --opt-sub-quad-attention --lowvram --no-half --disable-nan-check

Also make sure to install sd-webui-depth-lib .

**Reply by jjscholtes** [reply]
what did you change? with me its running on my cpu instead of my gpu (7900XT)

**Reply by ai-d-g** [reply]
I just reinstalled it stable Defussion WebUI,. installed deform and added those args and it worked.
